BusScheduleDB/3 reports.sql

```diff
@@ -0,0 +1,31 @@
+--AS You did not give very specific instructions for the first 2 so I just displayed a list according to instructions.
+--AS (You cannot do a group by on the route or the class if all the other columns are also just regular values not functions.)
+-- We need reports by bus route to be handed to the drivers.
```

Comment:
technically I would have wanted a page break when printing based on class or route but I guess we're not printing yet....